tests/reportBrowserAnalytics.test.ts

```diff
@@ -0,0 +1,38 @@
+import { reportBrowserAnalytics } from '../src/index';
+import { AnalyticsEventReporter } from '../src/AnalyticsEventReporter'; // replace with actual import path
+
+describe('reportBrowserAnalytics', () => {
+  let reportSpy: jest.SpyInstance;
+
+  beforeEach(() => {
+    // Mock window['analyticsEventPayload']
+    (global as any).window = {};
```

Comment:
Just to confirm, nothing we can do about these warnings?

Reply:
Unfortunately not. The alternatives cause actual errors. 

- Changing to unknown leads to `Object is of type 'unknown'.`

- Changing to never leads to `Property 'window' does not exist on type 'never'.ts(2339)`

- Removing the cast complety leads to `Type '{}' is not assignable to type 'Window & typeof globalThis'.

If we decide this is noisy we can also add an ignore comment to the top of the file in a future PR. 
`/* eslint-disable @typescript-eslint/no-explicit-any */`